libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -738,10 +738,27 @@ dictionary RecommendedFees {
     u64 minimum_fee;
 };
 
+[Enum]
+interface Amount {
+    Bitcoin(u64 amount_msat);
+    Currency(string iso4217_code, u64 fractional_amount);
+};
+
+dictionary LNOffer {
+    string bolt12;
+    sequence<string> chains;
+    string description;
+    string signing_pubkey;
+    Amount? amount;
+    u64? absolute_expiry;
+    string? issuer;
+};
+
 [Enum]
 interface InputType {
     BitcoinAddress(BitcoinAddressData address);
     Bolt11(LNInvoice invoice);
+    Bolt12(LNOffer offer);
```

Comment:
Let's call this Bolt12Offer?
https://github.com/breez/breez-sdk-greenlight/pull/1121#issuecomment-2468030319

Reply:
Done: a8a7057320aca4f5b03708a11de0ea66dc931d80